composer.json

```diff
@@ -99,6 +99,25 @@
         "psalm:baseline": "@php \"./.phive/psalm\" --set-baseline=psalm.baseline.xml",
         "psalm:cc": "@php \"./.phive/psalm\" --clear-cache"
     },
+    "scripts-descriptions": {
+        "ci": "Runs all dynamic and static code checks.",
+        "ci:composer:normalize": "Checks the formatting and structure of the composer.json.",
+        "ci:dynamic": "Runs all dynamic tests (i.e., currently the unit tests).",
+        "ci:php:fixer": "Checks the code stile with PHP CS Fixer.",
+        "ci:php:lint": "Lints the PHP files for syntax errors.",
+        "ci:php:md": "Checks the code size limits with PHPMD.",
+        "ci:php:psalm": "Checks the typing with Psalm.",
+        "ci:php:sniff": "Checks the code style with PHP_CodeSniffer.",
+        "ci:static": "Runs all static code analysis checks for the code and the composer.json.",
+        "ci:tests": "Runs all dynamic tests (i.e., currently the unit tests).",
```

Comment:
Not sure how this was intended, and perhaps beyond the scope of this PR, but if `ci:dynamic` and `ci:tests` have the same description and cannot be distinguished, we should remove one or other.

Reply:
Good point. I have added #1094 for this.